ci/docker/install/ubuntu_ccache.sh

```diff
@@ -24,6 +24,12 @@ set -ex
 pushd .
 
 apt update
+apt install -y \
```

Comment:
You could just remove `--no-install-recommends` from the call below - it would install most of the necessary packages. Actually, we had quite a discussion about where this installs should go: https://github.com/apache/incubator-mxnet/pull/11096 I'm not sure what was the outcome of it.